packages/launchpad-ledger/demo/node.js

```diff
@@ -0,0 +1,14 @@
+const demo = require("../build/demo/node");
+
+async function run() {
+  const accounts = await demo.getAccounts();
+  console.info("Accounts from Ledger device:");
+  console.table(accounts);
```

Comment:
Could this demo show a bunch of addresses (let's say account index 0, 1, 2) and the user selects which one to use?

Reply:
Good we checked because Ledger doesn't like more than one request at a time (e.g. from `Promise.all`)!